vortex-array/src/view.rs

```diff
@@ -116,7 +116,14 @@ impl ArrayView {
         let child = self.array_child(idx)?;
         let flatbuffer_loc = child._tab.loc();
 
-        let encoding = self.ctx.lookup_encoding(child.encoding())?;
+        let encoding = self
+            .ctx
+            .lookup_encoding(child.encoding())
+            .vortex_expect(&format!(
```

Comment:
Maybe make this function return a VortexResult?

Reply:
My thoughts as well. Is the context a user specified thing? If yes, this doesn’t seem like programmer error but user error ergo VortexResult.

Reply:
It is irrecoverable user error

Reply:
Ok, we can change this, it's just going to change how we handle validity in several places.